ui/components/app/confirm/info/row/row.tsx

```diff
@@ -84,7 +87,11 @@ export const ConfirmInfoRow = ({
       marginBottom={2}
       paddingLeft={2}
       paddingRight={2}
-      color={TEXT_COLORS[variant] as TextColor}
+      color={
+        labelEndChildren
+          ? getAlertTextColors(variant)
```

Comment:
If we're going to all this effort to decouple from the alerts, then should we just add an optional `color` property so we can specify this elsewhere and remove all the coupling?